packages/api/src/command/medical/organization/create-organization.ts

```diff
@@ -37,9 +41,41 @@ export const createOrganization = async (
   const fhirOrg = toFHIR(org);
   await upsertOrgToFHIRServer(org.cxId, fhirOrg);
 
-  if (org.type !== "healthcare_it_vendor") {
+  if (org.type === "healthcare_provider") {
+    // TODO Move to external/hie
     // Intentionally asynchronous
-    cwCommands.organization.create(org).catch(processAsyncError(`cw.org.create`));
+    cwCommands.organization
+      .createOrUpdate(cxId, {
+        oid: org.oid,
+        data: org.data,
+        active: org.cwActive,
+      })
+      .catch(processAsyncError(`cw.org.create`));
+
+    const { coordinates } = await getAddressWithCoordinates(org.data.location, cxId);
+    const address = org.data.location;
+    const addressLine = address.addressLine2
+      ? `${address.addressLine1}, ${address.addressLine2}`
+      : address.addressLine1;
```

Comment:
this logic is in a *lot* of places, let's put it in a function, and use it in all places

Reply:
Done.